src/handler-builder.ts

```diff
@@ -47,13 +47,22 @@ import * as testLab from './providers/testLab';
 export class HandlerBuilder {
   constructor() {}
 
+  /**
+   * Create a handler for HTTPS events.
+  
+   * `onRequest` handles an HTTPS request and has the same signature as an Express app.
+   * `exports.myFunction = functions.handler.https.onRequest((req, res) => { ... })`
+   *  This is identical to writing:
+   * `exports.myFunction = functions.https.onRequest((req, res) => { ... })`
+
+   * `onCall` declares a callable function for clients to call using a Firebase SDK.
+   * `exports.myFunction = functions.handler.https.onCall((data, context) => { ... })`
+   *  This is identical to writing:
```

Comment:
We may want to really explicitly clarify that we're comparing extensions handlers to "regular" function handlers, or maybe avoid the "identical" phrasing.

Even just changing "identical" to "similar" would make me feel better.  WDYT?